src/Header.messages.jsx

```diff
@@ -67,7 +67,7 @@ const messages = defineMessages({
     description: 'The aria label for the account menu trigger',
   },
   'header.label.account.menu.for': {
-    id: 'header.label.account.menu',
+    id: 'header.label.account.menu.for',
```

Comment:
this repo was previously not running `i18n_extract` as part of the CI, so this duplicate ID was never caught.

Reply:
good catch